Index: static/templates/chat/roll/power/power-buttons.hbs
===================================================================
--- static/templates/chat/roll/power/power-buttons.hbs	en
+++ static/templates/chat/roll/power/power-buttons.hbs	pl
@@ -1,9 +1,9 @@
 <div class="wrapper potency">
         {{#if result.isSuccess}}
         {{#if result.potency.options}}
-        <h3>Potency</h3>
+        <h3>Moc</h3>
         <div class="potency-allocation">
-                <h4>Spent: {{result.potency.spent}}/{{result.potency.available}}  <a data-action="resetPotency"><i class="fas fa-redo"></i></a> </h4>
+                <h4>Użyto: {{result.potency.spent}}/{{result.potency.available}}  <a data-action="resetPotency"><i class="fas fa-redo"></i></a> </h4>
                
         </div>
         <div class="potency-list">
\ No newline at end of file
